src/OrchardCore.Modules/OrchardCore.Users/Permissions.cs

```diff
@@ -8,16 +8,18 @@ public class Permissions : IPermissionProvider
     {
         public static readonly Permission ManageUsers = CommonPermissions.ManageUsers;
         public static readonly Permission ViewUsers = CommonPermissions.ViewUsers;
-
-        public static readonly Permission ManageOwnUserInformation = new Permission("ManageOwnUserInformation", "Manage own user information", new Permission[] { ManageUsers });
+        public static readonly Permission ManageOwnUserInformation = new("ManageOwnUserInformation", "Edit own user information", new Permission[] { ManageUsers });
 
         public Task<IEnumerable<Permission>> GetPermissionsAsync()
         {
             return Task.FromResult<IEnumerable<Permission>>(new List<Permission>
             {
                 ManageUsers,
-                ManageOwnUserInformation,
                 ViewUsers,
+                ManageOwnUserInformation,
+                CommonPermissions.ListUsers,
+                CommonPermissions.EditUsers,
+                CommonPermissions.DeleteUsers,
```

Comment:
@jtkech ListUsers is actually for listing users in the Admin UI. I think it should be called ListUsers. ViewUser could be for viewing someone profile "in display mode instead of edit".

Reply:
@jtkech we now also have a `Display` method to display the user to support `ListUsers` and `ViewUser`

Reply:
@MikeAlhayek

ViewUsers was not used for edit, it was already used to list/display users in the admin, just meaning that they are displayed but with the displayType: "SummaryAdmin".

No need to add a method just to say that we use it differently, but by just adding another `BuildDisplayAsync()` with another displayType.

I'm not against using `ListUsers` if it is really the better name that we should use, but then we loose the usage of the existing `ViewUsers`.

Reply:
@jtkech I think ListUsers is more meaningful for listing users in admin than ViewUsers. There is another PR for doing the same thing for Contents.

Reply:
Okay will look at this PR toonight